src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/Orderly.kt

```diff
@@ -224,29 +206,29 @@ class Orderly(val isReviewer: Boolean,
         JooqContext().use {
 
             val thisVersion = getReportVersion(name, version, it)
-            return getDatedChangelogForReport(thisVersion.report, thisVersion.date, it)
-
+            return getDatedChangelogForReport(thisVersion.name, Timestamp.from(thisVersion.date), it)
         }
 
     }
 
-    override fun checkVersionExistsForReport(name: String, version: String)
+    override fun getReportVersion(name: String, version: String): BasicReportVersion
```

Comment:
Renamed and changed to return the fetched object - this method will be moved to the report repo and used to get the basic version model when building up the more complex model in the logic class